pkgs/development/python-modules/tox/default.nix

```diff
@@ -8,15 +8,20 @@
 , virtualenv
 , setuptools-scm
 , toml
+, tomli
 , filelock
+, pythonAtLeast
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "tox";
   version = "3.26.0";
 
   buildInputs = [ setuptools-scm ];
-  propagatedBuildInputs = [ packaging pluggy py six virtualenv toml filelock ];
+  propagatedBuildInputs = [ packaging pluggy py six virtualenv filelock ]
+                          ++ lib.optional (pythonAtLeast "3.7" && pythonOlder "3.11") tomli
+                          ++ lib.optional (pythonOlder "3.7") toml;
```

Comment:
We don't have any Python older than 3.7 so this and toml can be removed. Or is this meant for Python2?

Reply:
I guess I can drop this:

```
╭─[~/git/nixpkgs]─[tox]
╰─ % nix-shell -p python2.pkgs.tox
error: freezegun-1.2.2 not supported for interpreter python2.7
(use '--show-trace' to show detailed location information)
```